backend_new/src/services/listing.service.ts

```diff
@@ -117,7 +117,16 @@ export class ListingService {
     this set can either be paginated or not depending on the params
     it will return both the set of listings, and some meta information to help with pagination
   */
-  async list(params: ListingsQueryParams) {
+  async list(params: ListingsQueryParams): Promise<{
+    items: ListingGet[];
```

Comment:
is adding the type explicitly here a required change or do you feel like this is a better pattern (making the return type as explicit as possible I mean)

Reply:
This change shouldn't theoretically be needed as the type can be inferred by the returned object. However, when running the tests in ci it was saying it was "unknown" type. There is potentially a configuration change that would make it better pick up types